Lib/distutils/command/bdist_rpm.py

```diff
@@ -341,10 +341,7 @@ def run(self):
         try:
             binary_rpms = []
             source_rpm = None
-            while True:
-                line = out.readline()
-                if not line:
-                    break
+            while (line := out.readline()):
```

Comment:
Let’s not make changes to distutils at this point (it’s deprecated and moved to setuptools).